AdminPortal/src/styles/Login.module.css

```diff
@@ -0,0 +1,147 @@
+.outer {
+    display: flex;
+    height: 600px;
+
+    /*align-items: baseline;*/
+
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+
+    gap: 10.275rem;
+}
+
+.LAAKTAlogo {
+    
+    width: 445.31px;
+    height: 445.31px;
+    margin-top: -40px;
+    /*position: absolute;
+    left: 147px;
+    top: 206px;*/
+}
+
+.formContainer {
+    
+    width: 30vw;
+    height: 496.64px;
+
+    /*position: absolute;
+    left: 756.71px;
+    top: 247.62px;*/
+
+    /*margin-top: auto;*/
+
+    background: #FFFFFF;
+    box-shadow: 0px 8.32361px 41.6181px rgba(0, 0, 0, 0.25);
+    border-radius: 27.7454px;
+}
+
+.emailPassword {
+    display: flex;
+    flex-direction: column;
+}
+
+.emailInput {
+    box-sizing: border-box;
+    background: #FFFFFF;
+    color: black;
+    border: 1.38727px solid #C4C4C4;
+    width: 23vw;;
+    height: 38.84px;
+    margin: 10vh 3.5vw 0px 3.5vw;
+    padding-left: 10px;
+    border-radius: 5px;
+}
+
+.input::placeholder{
+    position: absolute;
+    left: 3.33%;
+    right: 85.61%;
+    top: 17.86%;
+    bottom: 22.93%;
+
+    /* Body + Buttons */
+
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 17px;
```

Comment:
No components appear to be using the `input` class name. Did you mean `input::placeholder`?

I would also avoid using `position: absolute` as much as possible. You can simply set padding CSS on the input components to achieve this in a more adaptable way.

Reply:
I don't think it was being used anywhere, have removed it. Thanks for pointing that out!